app/models/host_genome.rb

```diff
@@ -46,6 +46,11 @@ def self.s3_bowtie2_index_path_default
     HostGenome::ERCC_PATH_PREFIX + HostGenome::S3_BOWTIE2_INDEX_FILE
   end
 
+  def self.all_without_metadata_field(name)
+    exists = joins(:metadata_fields).where(metadata_fields: { name: [name] })
+    where.not(id: [exists.pluck(:id).uniq])
```

Comment:
DO you nee pluck? What about - for a single query: 
```
exists = HostGenome.joins(:metadata_fields).where(metadata_fields: { name: [name] }).distinct
where.not(id: exists)
```
Not event sure you need the distinct... but maybe for perf.